GOVERNANCE.md

```diff
@@ -0,0 +1,67 @@
+
+As stewards of the project, Core Developers have additional duties beyond that of a maintainer. They are expected to represent the best interests of the project, above any employer or other company/organisational ties. The current Core Developers are:
+
+* Brian Brazil, Fabian Reinartz, Julius Volz
+
+New Core Developers are proposed by Core Developers on the prometheus-team@ mailing list, and voted on using the rules discussed in the voting section below. The current Core Developers are recognized as leaders of the Prometheus project as a whole, through their sustained and significant contributions across the project.
+
+## Core Developer Lead
+
+Core Developers are allowed to elect a Core Developer Lead who serves as the final say in any deadlock situation that can’t be resolved via a vote of the Core Developers or any maintainers. Instead of calling a vote, Core Developers and maintainers may always delegate a decision to the Core Developer Lead. However, if a vote is called, it takes precedence over the decision of the Core Developer Lead.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Committers and Maintainers
+
+Committer status (commit bit) is given to those who have made ongoing contributions to a given project. This is usually in the form of code improvements, notable work on documentation, organizing events or user support could also be taken into account. 
+
+Maintainers are committers who are expected to lead the project and serve as a point of conflict resolution amongst the committers.
+
+In order to become a committer or maintainer, an existing maintainer must nominate you on the public prometheus-developers@ mailing list. Once another maintainer (or core developer) seconds and there are no objections within a week, a core developer will add the nominee to the prometheus-team@ mailing list and give appropriate other access rights. If there is an objection which can't be resolved via consensus, a formal [vote](#heading=h.x907eu6orfvp) will occur. A maintainer or committer may resign by notifying the prometheus-team@ mailing list. A maintainer with no project activity for a year will be deemed to have resigned.
+
+## Voting
+
+The Prometheus project usually runs by informal consensus, however sometimes a formal decision must be made. All votes MUST be done on the prometheus-developers@ mailing list, and generally at least a week given for discussion between maintainers and for votes to take place. A majority of Core Developers must vote in favor for a vote to pass.
```

Comment:
:+1: this is a start, but I am generally for expanding the electorate (details TBD) sometime after we have this in place.

Depending on what you count as a "vote", we have had more than one vote in the past (lots of email chains with just +1s and -1s etc. in the past, though many didn't look like official votes because we didn't have anything in place yet anyways).